src/Microsoft.DotNet.Build.Tasks/UpdateVSConfigurations.cs

```diff
@@ -19,7 +19,7 @@ public class UpdateVSConfigurations : BuildTask
 
         private const string ConfigurationPropsFilename = "Configurations.props";
         private static Regex s_configurationConditionRegex = new Regex(@"'\$\(Configuration\)\|\$\(Platform\)' ?== ?'(?<config>.*)'");
-        private static string[] s_configurationSuffixes = new [] { "Debug|AnyCPU", "Release|AnyCPU" };
+        private static string[] s_configurationSuffixes = new [] { "Debug", "Release" };
```

Comment:
Just to confirm: every consumer of this is doing the switch so you don't need to keep the old format?

Reply:
I think it is OK to move this forward. If we need the old task for some reason we can fork it later or use a older branch.